client/state/timezones/reducer.js

```diff
@@ -27,8 +32,15 @@ export const byContinents = createReducer( {}, {
 	[ TIMEZONES_RECEIVE ]: ( state, actions ) => ( actions.byContinents )
 }, continentsSchema );
 
+export const requesting = createReducer( false, {
+	[ TIMEZONES_REQUEST ]: () => true,
```

Comment:
Oh I found it, we forgot to pass the action down the chain. Let's please update this:
```jsx
export const fetchTimezones = ( { dispatch }, action, next ) => {
//... wpcom request
    next( action );
}